src/Quiz.CSharp.Api/Services/QuestionService.cs

```diff
@@ -69,4 +65,23 @@ public async Task<List<QuestionResponse>> GetPreviewQuestionsAsync(
         var questions = await questionRepository.GetPreviewQuestionsAsync(collectionId, cancellationToken);
         return mapper.Map<List<QuestionResponse>>(questions);
     }
-} 
\ No newline at end of file
+
+    public async Task<QuestionResponse> UpdateQuestionAsync(int collectionId, int questionId, Dtos.Question.UpdateQuestion updateQuestion, CancellationToken cancellationToken)
+    {
+        var question = await questionRepository.GetSingleOrDefaultAsync(questionId, cancellationToken);
+        if (question is null)
+            throw new CustomNotFoundException($"{nameof(Question)} not found.");
```

Comment:
use null coalescing operator

```suggestion
        var question = await questionRepository.GetSingleOrDefaultAsync(questionId, cancellationToken)
            ?? throw new CustomNotFoundException($"{nameof(Question)} not found.");
```